README.md

```diff
@@ -108,7 +91,7 @@ Successfully built f29dd2ca5958
 
```

Comment:
Nit: for the docker output, I think that will have changed now that we no longer use tox.